src/transport/RendezvousSession.cpp

```diff
@@ -360,10 +326,6 @@ void RendezvousSession::OnRendezvousMessageReceived(const PacketHeader & packetH
         err = HandlePairingMessage(packetHeader, peerAddress, std::move(msgBuf));
         break;
 
-    case State::kNetworkProvisioning:
-        err = HandleSecureMessage(packetHeader, peerAddress, std::move(msgBuf));
-        break;
-
```

Comment:
Seems like `HandleSecureMessage` is unused now. It should be removed with all its dependencies.

Reply:
Removed now.